test/EFCore.PG.FunctionalTests/Query/RangeQueryNpgsqlTest.cs

```diff
@@ -501,7 +492,120 @@ public void UserDefined()
             }
         }
 
-        #endregion Tests
+        #endregion
+
+        #region Functions
+
+        [Fact]
+        public void RangeLowerBound()
+        {
+            using (RangeContext context = Fixture.CreateContext())
+            {
+                try
+                {
+                    var _ = context.RangeTestEntities.Select(x => x.Range.LowerBound).ToArray();
```

Comment:
Out of curiosity, does such variables make sense? As I know, there is no reason to make them since the compiler elides them.

Reply:
It keeps Rider/ReSharper from complaining about pure functions going unused, and IIRC `_` is viewable when stepping through the debugger.

But, yes, functionally they make no difference. But I would prefer to keep them for testing/debugging purposes.

Reply:
Okay, let's leave it for Rider.

Reply:
@austindrenski all of the Rider/Resharper complaints can be silenced via pragmas/omments, which seems like a better way to handle this.

Reply:
Actually scratch that, your way is better :)